src/Utils.ts

```diff
@@ -185,4 +185,12 @@ export const CL_DEVELOPER = 'ConversationLearnerDeveloper';
 export const UI_RUNNER_APPID = 'UIRunner_AppId'
 export const DEFAULT_MAX_SESSION_LENGTH = 20 * 60 * 1000;  // 20 minutes
 
+export const getSha256Hash: (id: string) => string = (id) => {
+    return crypto.createHash('sha256').update(id, 'utf8').digest('base64')
+}
+
+export const isRunningInCLUI: (context: BB.TurnContext) => boolean = (context) => {
```

Comment:
Very minor, but I dislike the mixing of capitalization techniques. I can be expected from different people who have different preferences but in the same review is strange, perhaps there was a reason why?

I prefer the convention Microsoft uses elsewhere, (not sure if it's only C# or not), where only the first letter of acronyms are capitalized because it helps people eyes separate adjacent acronyms.
```
ABCDEF
AbcDef  // Very clear these are things due to change in casing
```

Above you have:
```
getSha256Hash // Follows pattern which makes me expect
isRunningInClUi
```
However, below you have
```
isRunningInCLUI // which makes me expect
getSHA256Hash

```

Reply:
Id vs ID? Cl vs CL?

Reply:
In my opinion, `Id` and `Cl`